gulp/tasks/compare.js

```diff
@@ -59,7 +59,7 @@ gulp.task('compare', function (done) {
 
   function storeFailedDiffImage(testPath, data) {
     var failedDiffFilename = getFailedDiffFilename(testPath);
-    console.log('Storing diff image in ', failedDiffFilename);
+    console.log('Storing diff image in', failedDiffFilename);
```

Comment:
As long as we're at it, maybe the text should be shortened to: 
"       See: "

Essentially indent the file info to start with the error data value.